configs/config_sandbox/tf_version_debug/train-small-3class_tfversion_debug.yaml

```diff
@@ -6,6 +6,8 @@ train_config:
     model_parameters:
       backbone_name: 'vgg16'
       encoder_weights: Null   # Null (random initialization) or 'imagenet' (pre-training on ImageNet)
+      input_shape: [Null, Null, 1]
+      activation: 'sigmoid'
```

Comment:
@CarolinaFurtado , i guess the debug should also be updated alongside the most recent template? see `train-2dmodels-fpn-efficientnetb0.yaml` comments